<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>61</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>StockPro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stock per Material</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Batch number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Enter batch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scanButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Scan</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/icons/camera--plus.png</normaloff>../icons/icons/camera--plus.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Enter name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="placeholderText">
    <string>Type</string>
   </property>
   <item>
    <property name="text">
     <string>Appliances</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Peripherals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Systems</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Electronics </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Health &amp; Sports</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Networking</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Software &amp; Services</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Smart Home &amp; Security</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Others</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
